docs/pages/server-access/guides/azure-discovery.mdx

```diff
@@ -188,7 +196,18 @@ and replace the subscription in `assignableScopes` with your own subscription id
 
 (!docs/pages/includes/server-access/azure-assign-service-principal.mdx!)
 
-## Step 3/5. Install the Teleport Discovery Service
+## Step 3/6. Set up Managed Identities for Discovered Nodes
+
+Every Azure VM to be discovered must have a Managed Identity assigned to it
+with at least the `Microsoft.Compute/virtualMachines/read` permission. See the
+[Azure join method documentation](../../agents/join-services-to-your-cluster/azure.mdx#step-14-set-up-a-managed-identity)
+for instructions.
```

Comment:
I would recommend inlining these instructions within this guide. Having a user follow part of an otherwise unrelated guide could get confusing, and there's no need to direct the user away from this page.